extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -475,6 +523,14 @@ public static Token fromAudience(String... audience) {
         @ConfigItem
         public Optional<String> principalClaim = Optional.empty();
 
+        /**
+         * The time, in seconds, that tokens should be silently refreshed so that tokens with no active session at the
```

Comment:
I'd drop the word `silent` and would not even refer to the logout. If we have an expired ID Token then this refresh timeout will refresh all the token types for `web-app` applications. I can later use it for the pre-emtive refresh for the `service` applications too :-)

Reply:
For service types, it would not work because you don't have a refresh token. Only the bearer.

Reply:
Of course, got confused :-)

Reply:
@pedroigor I'd still remove `silent` - how else it can be refreshed ? :-) and drop a ref to the logout - we can still use it for the pre-emptive refresh for web-apps to refresh AT and ID tokens